lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.ipv.cri.address.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+public class RawSessionRequest {
+    @JsonAlias("client_id")
+    private String clientId;
+
+    @JsonAlias("request")
```

Comment:
As above, if we're ever going to re-serialize

Reply:
At present, there is no requirement to re-serialise